solidstart-tanstackquery-tailwind-modules/src/components/UserProfile/OrgList.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from '@solidjs/router';
+import { For } from 'solid-js';
+
+interface OrgListProps {
+  organizations: {
+    avatarUrl: string;
+    login: string;
+  }[];
+}
+
+function OrgList(props: OrgListProps) {
+  const navigate = useNavigate();
+  return (
+    <div class="mt-5 border-t border-gray-200">
+      <h2 class="my-2 pt-2 text-gray-800 font-bold">Organizations</h2>
+      <div data-testid="profile page orgs" class="flex flex-wrap space-x-2">
+        <For each={props.organizations}>
+          {(props) => (
+            <a
```

Comment:
Doesn't the A or Link tag from solidjs/router work?

Try that first.
The Link and A tag from solid state seems faulty

Reply:
Caught and fixed